app/controllers/review_item/reject.js

```diff
@@ -46,120 +46,159 @@ export default Ember.Controller.extend({
   selectedId: Ember.computed("rejectionReasonId", {
     get: function() {
       this.set("isBlank", false);
-      var reasonId = this.get('rejectionReasonId');
-      if(reasonId) { return reasonId; }
-      else {
-        if(this.get("rejectReason") && this.get("rejectReason").length > 0) {
+      var reasonId = this.get("rejectionReasonId");
+      if (reasonId) {
+        return reasonId;
+      } else {
+        if (this.get("rejectReason") && this.get("rejectReason").length) {
           return "-1";
         }
       }
     },
     set: function(key, value) {
       this.set("isBlank", false);
-      this.set('noReasonSelected', false);
+      this.set("noReasonSelected", false);
       return value;
     }
   }),
 
-  rejectionOptions: Ember.computed(function(){
-    return this.store.peekAll('rejection_reason').sortBy('id');
+  rejectionOptions: Ember.computed(function() {
+    return this.store.peekAll("rejection_reason").sortBy("id");
   }),
 
-  cannotSave(){
-    var pkgs = this.store.peekRecord("item", this.get("itemId")).get("packages");
-    if(pkgs && pkgs.length > 0 && (pkgs.get("firstObject.hasAllPackagesDesignated") || pkgs.get("firstObject.hasAllPackagesDispatched"))){
-      this.get('messageBox').alert(this.get("i18n").t('designated_dispatched_error'));
+  cannotSave() {
+    var pkgs = this.store
+      .peekRecord("item", this.get("itemId"))
+      .get("packages");
+    if (
+      pkgs &&
```

Comment:
Assign this condition to any variable with some meaningful name, so that i should be easy to understand.

Reply:
This is not used anywhere else so why to assign it to the variable? 🤔